.github/workflows/ios-release.yaml

```diff
@@ -30,6 +30,12 @@ jobs:
           toolchain: stable
           target: aarch64-apple-ios
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: aarch64-apple-ios-sim
```

Comment:
can we merge this block with the previous ones? having multiple target that is

Reply:
I tried space separated and comma separated, but this didn't work - any other suggestions?

https://github.com/c4dt/lightarti-rest/actions/runs/3428008325/jobs/5711710484#step:5:50
https://github.com/c4dt/lightarti-rest/actions/runs/3428000679/jobs/5711692270#step:5:50

Reply:
According to https://github.com/actions-rs/toolchain it's a string.

Reply:
hum, no, and I don't see a way to actual expand multiples target [in the code](https://github.com/actions-rs/core/blob/master/src/commands/rustup.ts#L146), nevermind